src/api/mutable_data_entries.js

```diff
@@ -66,8 +65,8 @@ module.exports.get = (entriesHandle, keyName) => {
  *          console.log('Version: ', v.version);
  *       }).then(_ => console.log('Iteration finished'))
  *    );
- **/
-module.exports._with_cb_forEach = (entriesHandle) => {
+*/
+module.exports._with_cb_forEach = (entriesHandle) => { // eslint-disable-line max-len, no-underscore-dangle
```

Comment:
Why do we need the eslint exception here since it was ok before?

Reply:
When running `yarn lint` or `npm run lint` there are many lint errors, so I'm not sure what was happening before.